### ISSUE TITLE
`First_Terminal, Second_Terminal` use unusual separator

### ISSUE THREAD

**Issue by VladimirAlexiev** [Sveino/Inst4CIM-KG#107]
Slightly related to #106, I looked for correspondence between localnalname and label.
```sparql
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX afn: <http://jena.apache.org/ARQ/function#>
select * {
    ?x rdfs:label ?label
    bind(afn:localname(?x) as ?localname)
    bind(replace(?localname,".*[_.]","") as ?lastPart)
    filter(str(?label) != ?lastPart && str(?label) != ?localname)
}
```
I found only 2 terms that use "_" as separator between words. I think that for consistency with the several thousand other terms, you should remove it

|   | x                                    | label                  | localname                         | lastPart   |
|---|--------------------------------------|------------------------|-----------------------------------|------------|
| 1 | cim: MutualCoupling.First_Terminal  | "First_Terminal"@ en  | "MutualCoupling.First_Terminal"  | "Terminal" |
| 2 | cim: MutualCoupling.Second_Terminal | "Second_Terminal"@ en | "MutualCoupling.Second_Terminal" | "Terminal" |

**Comment by Sveino** [comment]
Yes. This is a CIM18 issue. We should follow the CamelCase notation.
